.github/workflows/coverage.yml

```diff
@@ -0,0 +1,46 @@
+name: Coverage Report
+
+on:
+  push:
+    branches: [ "develop" ]
```

Comment:
I'm not sure we need to run after merging into develop because there's no chance of us seeing the report. (Unless the `develop` report is used to compare to PR reports.)